splice_machine/src/main/java/com/splicemachine/derby/stream/function/NLJoinFunction.java

```diff
@@ -108,8 +109,9 @@ private void loadBatch() throws StandardException {
                 currentOperationContext = result.getFirst();
                 rightSideNLJIterator = result.getSecond();
                 leftRow = currentOperationContext.getOperation().getLeftOperation().getCurrentRow();
-                leftRowLocation = currentOperationContext.getOperation().getLeftOperation().getCurrentRowLocation();
+                leftRowLocation = new HBaseRowLocation(leftRow.getKey());
                 operationContext.getOperation().getLeftOperation().setCurrentRow(getLeftLocatedRow());
+                operationContext.getOperation().getLeftOperation().setCurrentRowLocation(getLeftRowLocation());
```

Comment:
It's for correctness of delete over nested loop join. row location is still needed and needs to be populated.